packages/system/src/Item/Components/StockItemSelectModal/StockItemSelectModal.tsx

```diff
@@ -126,17 +151,23 @@ export const StockItemSelectModal = ({
           <AutocompleteList
             options={options}
             loading={loading}
-            height={375}
+            height={365}
             width={600}
             disableCloseOnSelect
             multiple
             getOptionLabel={option => `${option.code} ${option.name}`}
+            filterOptions={(options, state) =>
+              options.filter(option =>
+                RegexUtils.matchNameOrCode(option, state.inputValue)
+              )
+            }
```

Comment:
Will need to update function name as a result of your other PR changes.

Just out of interest -- when you merge a branch to `main` do you have any pre-merge checks? Cos this branch is currently valid, and `main` is currently valid, but once you merge this into `main`, there will be compile errors due to the function name change from the other branch.

Reply:
no - just the pre-commit check. It's theoretically possible that the merge makes a hash of things - but hasn't been an issue yet.